### PR TITLE
fix: fixing docs setup

### LINKED ISSUE #7792: [Docs] Tidying Up Meshery documentation
#### Current State
There are little issues in the Docs that have to be addressed
1. All content under the guide is not indexed in the sidebar
2.  Broken link under for[ mesheryctl command reference](https://docs.meshery.io/reference/mesheryctl) under [configuration-management](https://docs.meshery.io/guides/configuration-management)
3. App [import](https://docs.meshery.io/reference/mesheryctl/app/import) subcommand is not indexed in the reference page 
4.  Update the link for [installation methods ](https://docs.meshery.io/installation) under [quick start](https://docs.meshery.io/installation/quick-start), currently it directs to supported platforms
5.  Correct the command for setting up the env variable for [Windows](https://docs.meshery.io/installation/platforms/windows#4--install-a-kubernetes-cluster) 
 from `export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"` to the following 
` $env:KUBECONFIG = "$(k3d.exe kubeconfig get 'k3s-default')"`

#### Desired State

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by SimardeepSingh1450**
**Notes for Reviewers**

This PR fixes #7792 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 
#### Changes :
1. Fixed the broken link under for[ mesheryctl command reference](https://docs.meshery.io/reference/mesheryctl) under [configuration-management](https://docs.meshery.io/guides/configuration-management)
2. Added App [import](https://docs.meshery.io/reference/mesheryctl/app/import) subcommand in the sidebar of reference.
3. Updated the link for [installation methods ](https://docs.meshery.io/installation)under [quick start](https://docs.meshery.io/installation/quick-start) correctly, initially it directed to supported platforms.
4. Corrected the command for setting up the env variable for [Windows](https://docs.meshery.io/installation/platforms/windows#4--install-a-kubernetes-cluster)
from `export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"` to the following
 `$env:KUBECONFIG = "$(k3d.exe kubeconfig get 'k3s-default')"`

### CODE REVIEW

File: docs/pages/installation/windows.md [hunk 1]
```diff
@@ -131,31 +136,33 @@ The Docker Desktop application for Windows includes a comprehensive set of tools
...
 Once Docker is installed, the next step will be to install a Kubernetes cluster.
-Under "Settings" in the Docker Desktop application, enable *Kubernetes*.
+Under "Settings" in the Docker Desktop application, enable _Kubernetes_.
 In this how-to, [K3d](https://github.com/rancher/k3d) will be used as it relies only on Docker.
 
   <pre class="codeblock-pre">
   <div class="codeblock"><div class="clipboardjs">curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
 k3d cluster create
-export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"</div></div>
+$env:KUBECONFIG = "$(k3d.exe kubeconfig get 'k3s-default')"</div></div>
```
Comment (thisiskaransgit) [inline docs/pages/installation/windows.md:145]: Here, we are using bash script, we only needed the change in line 153, please revert this one

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@SimardeepSingh1450,
- App import has to be added in [reference](https://docs.meshery.io/reference/mesheryctl) page under the [application management table](https://docs.meshery.io/reference/mesheryctl#cloud-native-application-management) command table. Not in the sidebar.
- Also while pushing the changes please revert the changes made in gemfiles

**Comment by l5io** [comment]
🚀 Preview for commit 29e04e0fbecb2a8a33eb5b057dcc2da8c54052f2 at: https://64901f3d44734c3f234f158f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d7279bf24058117b0af245dca0358b7136016ee5 at: https://649026d2c41d51480a59bac9--meshery-docs-preview.netlify.app

**Comment by SimardeepSingh1450** [comment]
@thisiskaransgit can you please tell from where i can change the [reference page](https://docs.meshery.io/reference/mesheryctl) content.

**Comment by thisiskaransgit** [comment]
> @thisiskaransgit can you please tell from where i can change the [reference page](https://docs.meshery.io/reference/mesheryctl) content.

@SimardeepSingh1450  Refer to this [docs](https://docs.meshery.io/project/contributing/contributing-cli#adding-newremoving-existing-commands-in-the-reference-index)

**Comment by l5io** [comment]
🚀 Preview for commit 2fcc3fd7aca7803bad72eb15b79d602d67987fb4 at: https://6490439144734c55f74f146a--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c2609047016c8799ea97a1885d1ca5084821985a at: https://649045500335765c2868d9dd--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 639f8b026874d2e5fd860f226d9538d81d5563b3 at: https://64904a6e11e1f660d864fadb--meshery-docs-preview.netlify.app

**Comment by SimardeepSingh1450** [comment]
@thisiskaransgit why is it behaving like this ?
image

**Comment by vishalvivekm** [comment]
@SimardeepSingh1450 
Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

File: docs/_data/mesheryctlcommands/cmds.yml
Comment (thisiskaransgit) [inline docs/_data/mesheryctlcommands/cmds.yml:1]: Please do not change the coding style.

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Comment by thisiskaransgit** [comment]
> @thisiskaransgit can you please tell why is import source-type flag under app behaving like this ? image


@SimardeepSingh1450 
It does not have to do with the workflow, Increase the `rowspan` for Application Management Table, and it will work fine

File: docs/_data/mesheryctlcommands/cmds.yml
Comment (thisiskaransgit) [reply docs/_data/mesheryctlcommands/cmds.yml:1]: Your extensions might have overridden the default coding style, changing the code style makes the reviewing work difficult and is counted as unwanted, when pushed without prior discussions. Remember it for the next time. Kindly Revert it.

**Comment by thisiskaransgit** [comment]
@SimardeepSingh1450, The rest looks good, incorporate the above requests and we're good to go 👍🏼

**Comment by thisiskaransgit** [comment]
@SimardeepSingh1450, What's your progress?

File: docs/pages/reference/mesheryctl-commands.md
Comment (leecalcote) [inline docs/pages/reference/mesheryctl-commands.md:1]: @SimardeepSingh1450 thanks for this PR. will you please remove all other changes outside of this file and [line 154](https://github.com/meshery/meshery/pull/7890/files#diff-edb042f2d27d66af3fb28f3e3a3b4651af11e9844f1f7f087f76d6b8651f66dbR154) of the windows.md file?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@SimardeepSingh1450 ?

**Comment by leecalcote** [comment]
merge conflicts resolved

**Comment by l5io** [comment]
🚀 Preview for commit a75745e2ab68146d082ddb774651d67843e416b4 at: https://64a407fa4016ad008948ab2d--meshery-docs-preview.netlify.app

**Comment by Chadha93** [comment]
@SimardeepSingh1450 Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by leecalcote** [comment]
@SimardeepSingh1450 Will you be incorporating this feedback?

**Comment by thisiskaransgit** [comment]
Closed as completed via #8128

**Pull request closed without merging**